Spigot-API-Patches/0283-Can-Fly-Fall-Damage.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TreyRuffy <TreyRuffy@users.noreply.github.com>
+Date: Sat, 13 Mar 2021 13:55:27 -0700
+Subject: [PATCH] Can Fly Fall Damage
+
+
+diff --git a/src/main/java/org/bukkit/entity/HumanEntity.java b/src/main/java/org/bukkit/entity/HumanEntity.java
+index f0e0710fef5a3e0b722ece7ccf89c3d0f88f8f0f..70b2617dc9ef4cdaa04ad499f7d3211c50e42315 100644
+--- a/src/main/java/org/bukkit/entity/HumanEntity.java
++++ b/src/main/java/org/bukkit/entity/HumanEntity.java
+@@ -576,6 +576,25 @@ public interface HumanEntity extends LivingEntity, AnimalTamer, InventoryHolder
+      * @param sign The sign to open
+      */
+     void openSign(@NotNull org.bukkit.block.Sign sign);
++
++    // Add method to enable fall damage if you have the allowFlight attribute
++    /**
++     * Allows you to enable fall damage while {@link Player#getAllowFlight()} is true
++     * <br>
++     * Get with {@link #getEnableFallDamageWhileAllowFlight()}
++     *
++     * @param fallDamage Enables fall damage with allowFlight is true
```

Comment:
This seems like the best place to refer to the allowFlight. I question whether the javadocs are overly repetitive.